src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -143,6 +143,17 @@ public static void ForAll_OperationCanceledException_PreCanceled(LabeledOperatio
         [Theory]
         [MemberData(nameof(UnaryOperators))]
         [MemberData(nameof(BinaryOperators))]
+        public static void ForEach_OperationCanceledException_PreCanceled(LabeledOperation source, LabeledOperation operation)
+        {
+            CancellationTokenSource cs = new CancellationTokenSource();
+            cs.Cancel();
```

Comment:
Ignore this comment.  In a subsequent commit you isolate all of these to a single location, and it's fine to use to verify that the same exact token is used.